src/wrappers/TaxonConceptWrapper.js

```diff
@@ -62,6 +63,24 @@ class TaxonConceptWrapper {
     return undefined;
   }
 
+  /**
+   * Return the nomenclatural code of this taxon concept as a string.
+   */
+  get nomenCode() {
+    if (has(this.tunit, 'hasName')) return new TaxonNameWrapper(this.tunit.hasName).nomenclaturalCode;
+
+    return undefined;
+  }
+
+  /**
+   * Return the nomenclatural code of this taxon concept as an object.
+   */
+  get nomenCodeAsObject() {
```

Comment:
This is a weird design. If there's a native object representation, shouldn't then `nomenCode()` return that object, and the objects should have an `toString()` or `asString()` method if someone wanted a flat representation?

Reply:
I think the confusing thing here is that the `*AsObject` getters here don't actually return objects: they return dicts containing more detailed information about that particular nomenclatural code. I've renamed them to `*Details` (as in `nomenCodeDetails`) in 967cfc012e15 -- is that clearer?

These are convenience methods for accessing the static method `TaxonNameWrapper.getNomenCodeDetails(nomenCodeURI)` via the TaxonNameWrapper as that made it a lot easier to access them in Vue.js. Outside of something like that, it probably makes more sense to call that static method directly.

Actually, these are convenience methods for accessing the nomenclatural code from the taxon *concept* -- so I could further get rid of `TaxonConceptWrapper.nomenCodeDetails` -- you can use `TaxonConceptWrapper.nomenCode` to access the nomenclatural code on the underlying taxon name, and then use `TaxonNameWrapper.getNomenCodeDetails(nomenCodeURI)` to retrieve the details about that nomenclatural code. Do you think that would be clearer?

Reply:
I think so. Though is there a difference between `nomenCode` and `nomenCodeURI`? If not, use one or the other but not both.

Reply:
Fair. I've standardized it to `nomenCode` in b4316db3bc.